test/support/machine.go

```diff
@@ -0,0 +1,32 @@
+package support
+
+import (
+	"github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
+)
+
+func GetMachineSets(t Test) ([]machinev1beta1.MachineSet, error) {
+	ms, err := t.Client().Machine().MachineV1beta1().MachineSets("openshift-machine-api").List(t.Ctx(), metav1.ListOptions{})
+	t.Expect(err).NotTo(gomega.HaveOccurred())
+	return ms.Items, err
+}
+
+func Machines(t Test, machineSetName string) func(g gomega.Gomega) []machinev1beta1.Machine {
+	return func(g gomega.Gomega) []machinev1beta1.Machine {
+		machine, err := t.Client().Machine().MachineV1beta1().Machines("openshift-machine-api").List(t.Ctx(), metav1.ListOptions{LabelSelector: "machine.openshift.io/cluster-api-machineset=" + machineSetName})
```

Comment:
Could passing the label selector as argument instead be useful?

Reply:
I'm not sure if we can access the machine label easily in the test to pass it in. Could this work, or what would be the benefit of passing it as an argument? maybe it can be one variable instead of 2 also since these shouldn't change.
```
labelSelectorPrefix = "machine.openshift.io/cluster-api-machineset="
machine, err := t.Client().Machine().MachineV1beta1().Machines(namespaceToList).List(t.Ctx(), metav1.ListOptions{LabelSelector: labelSelectorPrefix + machineSetName})
```